oio/api/base.py

```diff
@@ -20,20 +20,14 @@
 from oio.common.json import json as jsonlib
 
 from oio.common.http_urllib3 import urllib3, get_pool_manager, \
-    oio_exception_from_httperror
+    oio_exception_from_httperror, URLLIB3_POOLMANAGER_KWARGS,
```

Comment:
```suggestion
    oio_exception_from_httperror, URLLIB3_POOLMANAGER_KWARGS, \
```

Reply:
Good catch, I didn't test this part as I found out later the swift gateway uses only clientmanager